docs/reference/nanodet.md

```diff
@@ -0,0 +1,289 @@
+## nanodet module
+
+The *nanodet* module contains the *NanodetLearner* class, which inherits from the abstract class *Learner*.
+
+### Class NanodetLearner
+Bases: `engine.learners.Learner`
+
+The *NanodetLearner* class is a wrapper of the Nanodet object detection algorithms based on the original
+[Nanodet implementation](https://github.com/RangiLyu/nanodet).
+It can be used to perform object detection on images (inference) and train All predefined Nanodet object detection models and new modular models from the user.
+
+The [NanodetLearner](../../src/opendr/perception/object_detection_2d/nanodet/nanodet_learner.py) class has the
+following public methods:
+
+#### `NanodetLearner` constructor
+```python
+NanodetLearner(self, model_to_use, iters, lr, batch_size, checkpoint_after_iter, checkpoint_load_iter, temp_path, device,
+               weight_decay, warmup_steps, warmup_ratio, lr_schedule_T_max, lr_schedule_eta_min, grad_clip)
+```
+
+Constructor parameters:
+
+- **model_to_use**: *{"EfficientNet-Lite0_320", "EfficientNet-Lite1_416", "EfficientNet-Lite2_512", "RepVGG-A0_416",
+  "t", "g", "m", "m-416", "m-0.5x", "m-1.5x", "m-1.5x-416", "plus-m_320", "plus-m-1.5x_320", "plus-m_416",
+  "plus-m-1.5x_416", "custom"}*\
```

Comment:
I would also add what it the default value.

Reply:
Thank you for your suggestion.

Reply:
Thank you for your suggestion.